test/backend/inference_backend.py

```diff
@@ -77,13 +77,17 @@ def get_test_models():
         # ==ARCH== cpu
 
         # ==OP== Abs
+        # ==MIN== 6
+        #min=1/max=13
```

Comment:
I thought the idea was that here in the "code" we'd have the "true" Op Versions we support. Then in the documents, we show the min model OpSet we support and the and the maximum Opset that can works with the Op (or * if we support latest). 

Also to make things clearer and easier to doc, I'd recommend we rename ==MAX== with something like ==UNSUPPORTED==. (clearer when used with `-1` in doc generator see further below)

So given the `Fake` Op example. Say we support Op V 1 up to Op V 10. Then in OpSet 19 there's a new `Fake` version 19 that we don't support. Here in the "code" that would look like:
```
# ==MIN== 1
# ==UNSUPPORTED== 19
```
(Note: There'd be no UNSUPPORTED line if we support the latest Op Version available in the latest OpSet)

Then in the document generator it would look something like: 
```
min_opset_default="6"
max_opset_default="*"

    if (==MIN==):  // No ==MIN== implies unsupported so no version to show
        opset_min = max(==MIN==, min_opset_default)
        opset_max = ==UNSUPPORTED== -1 if ==UNSUPPORTED== else max_opset_default
```

Reply:
@cjvolzka I explained it does not work like the way you are showing above.... what I specify in the python script WILL be generated in the document view. There is ONLY one view. That is why I put in the comments for the python script min=1/max=19 as a reference. 

When I said should we still do the `unsupported` and etc. everyone said no. What we agreed on is the minimum being set to 6 which it is and 18 as the max. For a scenario when the minimum is 18 then the max would be `*` to indicate that we are at the current version. We all agreed that we are supporting things from an opset perspective and not version. Hence why I changed the subheading back to make things clear. We agreed to change it from min/ max version to min/max opset. 

I think it should be one way or the other not both. Either a) we set the minimum to 6 and the max to 18 and use `*` for everything that is already at the max version b) just go with 1 but I think this is becoming overly complicated.......I think simplicity is key here. 

From our conversation with @AlexandreEichenberger he stated "we should not mention support for opsets we do not support". So having 1 does not really make sense which you agreed to.